interfaces/mediaont-api.idl

```diff
@@ -0,0 +1,164 @@
+// GENERATED CONTENT - DO NOT EDIT
+// Content of this file was automatically extracted from the
+// "Metadata API for Media Resources 1.0" spec.
+// See: https://www.w3.org/TR/mediaont-api-1.0/
```

Comment:
This is a spec which I'd consider not part of the web platform, despite using IDL. AFAICT, the IDL is disconnected from the rest of the web platform, in particular it doesn't integrate with HTMLMediaElement in any way, or any other media-related API AFAICT. But more importantly, this will never be implemented by any browser.

Reply:
Filed https://github.com/tidoust/reffy-reports/issues/7.